### PR TITLE
Godep update in FAQ

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @MaciekPytel @bskiba @aleksandra-malinowska @wasylkowski

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -506,3 +507,26 @@ required to activate them:
    https://github.com/kubernetes/autoscaler/pull/74#issuecomment-302434795).
 
 We are aware that this process is tedious and we will work to improve it.
+
+### How can I update CA dependencies (particularly k8s.io/kubernetes)?
+
+CA depends on `k8s.io/kubernetes` internals as well as the "official" k8s.io libs like 
+`k8s.io/apimachinery`. However `k8s.io/kubernetes` has its own/newer version of this libraries
```
Comment (bskiba) [inline cluster-autoscaler/FAQ.md]: "versions of these libraries"

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -506,3 +507,26 @@ required to activate them:
...
+(in a `staging` directory) which may not always be compatibile with what has been published.
+This leads to various conflicts that are hard to resolve in a "proper" way. So until a better solution 
+is proposed (or we stop migrating stuff between k8s.io/kubernets and other projects on a daily basis),
+the followin hack has to be used to make the things easier to handle.
```
Comment (bskiba) [inline cluster-autoscaler/FAQ.md]: s/followin/following/

File: cluster-autoscaler/FAQ.md (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: done.

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: done

**Pull request merged**